web/src/features/landingPage/components/Sidebar.tsx

```diff
@@ -15,8 +16,8 @@ import Typography from '@mui/material/Typography'
 import useMediaQuery from '@mui/material/useMediaQuery'
 import SidebarToolList from 'features/landingPage/components/SidebarToolList'
 import Subheading from 'features/landingPage/components/Subheading'
-import { ReactComponent as MsTeamsIcon } from 'features/landingPage/images/msTeams.svg'
-import { ReactComponent as MiroIcon } from 'features/landingPage/images/miro.svg'
+import MsTeamsIcon from 'features/landingPage/images/msTeams.svg?react'
```

Comment:
Interesting syntax! I haven't seen this before. Nice!

Reply:
it's a little freaky eh